.github/workflows/e2e.yml

```diff
@@ -83,6 +83,9 @@ jobs:
       NODE_ENV: test
       VERSION_INFO_GIT_REF: ${{ github.ref }}
       DC_ACCOUNTS_DIR: ../../e2e-tests/data/accounts
+      DC_CHATMAIL_SERVER: ${{ secrets.CI_CHATMAIL_SERVER }}
```

Comment:
Sooo, are we supposed to hide the CI Chatmail server from others? They can't run tests locally then.

Also, I believe we can't really hide this (and the non-Chatmail server), because it's exposed in the UI, namely in "account and password" settings anyway as tests run. If a test fails, the Playwright report might contain a video where this can be seen.
And the email also exposed in the test artifact output. And in the test logs (I am not sure why it didn't get auto-redacted).

Reply:
Well for the server URLs the idea is more to make them configurable without touching the code/tests. Only the token is a "real" secret.

But you are right, the server names could just be variables then.